polynote-frontend/polynote/repr_ui.js

```diff
@@ -58,7 +58,7 @@ function renderData(dataType, data) {
     // TODO: nicer display
     let value = '';
     if (dataType instanceof ArrayType || dataType instanceof StructType) {
-        value = JSON.stringify(item);
+        value = JSON.stringify(data);
```

Comment:
😳 I've been spoiled by static types

Reply:
You and me both... https://www.typescriptlang.org/ 😉 🤔